toast-tk-dao/src/main/java/io/toast/tk/dao/service/dao/access/team/TeamDaoService.java

```diff
@@ -0,0 +1,38 @@
+package io.toast.tk.dao.service.dao.access.team;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.github.jmkgreen.morphia.query.Query;
+import com.google.inject.Inject;
+import com.google.inject.assistedinject.Assisted;
+import com.google.inject.name.Named;
+
+import io.toast.tk.dao.domain.impl.common.IServiceFactory;
+import io.toast.tk.dao.domain.impl.team.TeamImpl;
+import io.toast.tk.dao.domain.impl.team.UserImpl;
+import io.toast.tk.dao.service.dao.common.AbstractMongoDaoService;
+import io.toast.tk.dao.service.dao.common.CommonMongoDaoService;
+import io.toast.tk.dao.service.init.DbStarter;
+
+public class TeamDaoService extends AbstractMongoDaoService<TeamImpl> {
+
+	public interface Factory extends IServiceFactory<TeamDaoService>{
+	}
+
+	@Inject
+	public TeamDaoService(
+		final DbStarter starter,
+		final CommonMongoDaoService cService,
+		final @Named("default_db") String default_db,

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AS00117)